core/src/scene/styleMixer.cpp

```diff
@@ -201,9 +201,6 @@ void StyleMixer::mergeFieldTakingLast(std::string key, Node target, const std::v
 void StyleMixer::mergeMapFieldTakingLast(std::string key, Node target, const std::vector<Node>& sources) {
```

Comment:
I'm not convinced we should generally rely that much on that,
quick bench shows it's much faster to pass by const ref, even with optimization on:
https://gist.github.com/karimnaaji/d58afb771de5edfc98a1

```
~/Desktop/cpp-benchs  master ✗                                                                                                                                               80d ⚑  
> g++ const_string_vs_value_string.cpp -O3 -std=c++11

~/Desktop/cpp-benchs  master ✗                                                                                                                                               80d ⚑  
> ./a.out                                            
function_a: 0.002ms         # const ref 
function_b: 68.776ms       # copy
function_b-move: 2.86ms # move
```